openshift/dev.env

```diff
@@ -0,0 +1,9 @@
+#----- OpenShift Settings
+
+MEMORY_REQUEST=512Mi
+MEMORY_LIMIT=2Gi
+CPU_REQUEST=500m
+CPU_LIMIT=1
+VOLUME_CAPACITY=5Gi
+SC_MONGO=gluster-block
```

Comment:
change for `netapp-file-standard` gluster can't be provisioned anymore

Reply:
There is an existing bug realted to lun id in netapp-file-standard. When I hit that, I raised it to the devops team (as well as in scrum ;) ). As per devOps team's recommendation I have changed it to gluster-block after getting gluster-block quota. Also, this is currently deployed and working in xdwiwd-dev namespace in openshift.

Reply:
gotchya :) can we add a note on that in as docs. This i suspect is a temporariy solution. Perhaps we can leave a note on the github issue tracking the net app bug

Reply:
also fyi, when you swap back to netapp, you should consider trying netapp-block and not netapp-file, as block is generally better for databases!